python/langsmith/run_trees.py

```diff
@@ -318,7 +318,9 @@ def patch(self) -> None:
         """Patch the run tree to the API in a background thread."""
         if not self.end_time:
             self.end()
-        attachments = self.attachments
+        attachments = {
+            a: v for a, v in self.attachments.items() if isinstance(v, tuple)
+        }
```

Comment:
This was done strictly for typing

Reply:
is v ever not a tuple?

Reply:
If so might be worth doing in a for loop and raising an error if not. I think would be unexpected. But mypy probably knows better than me, so there might be a case we're not handling here

Reply:
v is never not a tuple here, this is just because of the union types I believe and it thinks that something could be passed here that is never passed